test/system/scenarios/trailing-stop-buy-relative.js

```diff
@@ -0,0 +1,53 @@
+const run = (trader1, trader2, trader3) => {
+  console.log('****************************************************************');
+  console.log('** Scenario 5: Trailing Stop Buy Relative                             **');
+  console.log('****************************************************************');
+  return trader1.init_state(trader1.config)
+    .then(() => trader2.init_state(trader2.config))
+    .then(() => trader3.init_state(trader3.config))
+    .then(() => trader1.placeLimitOrderSpread(0.25, 0.5, 5, 0.01))
+    .then(() => trader2.placeTrailingStopOrder('B', 0.26, 'R', 0.05))
+    .then(() => trader3.placeLimitOrder('B', 1, 0.26))
+    .delay(1500)
+    .then(() => trader1.getMyTrades())
+    .then(() => trader3.getMyTrades())
```

Comment:
Need to look in trader1.trades and trader3.trades for a trade with quantity 0.5 and price 0.26